UI/qt-display.cpp

```diff
@@ -197,6 +197,8 @@ bool OBSQTDisplay::nativeEvent(const QByteArray &, void *message, long *)
 	case WM_DISPLAYCHANGE:
 		OnDisplayChange();
 	}
+#else
+    (void)message;
```

Comment:
Use the UNUSED_PARAMETER macro instead.

Reply:
> Use the UNUSED_PARAMETER macro instead.

Fix in last update (also fix clang-format check)